src/Shared/runtime/Http3/QPack/H3StaticTable.Http3.cs

```diff
@@ -48,7 +48,7 @@ public static bool TryGetStatusIndex(int status, out int index)
         // TODO: just use Dictionary directly to avoid interface dispatch.
         public static IReadOnlyDictionary<HttpMethod, int> MethodIndex => s_methodIndex;
 
-        public static HeaderField GetHeaderFieldAt(int index) => s_staticTable[index];
+        public static ref HeaderField Get(int index) => ref s_staticTable[index];
```

Comment:
Can this be `ref readonly`?

Reply:
Maybe. This could also apply to HPACK code.

I'll create a follow up issue to investigate this and check what impact it has on HPACK/QPACK perf.

Reply:
https://github.com/dotnet/aspnetcore/issues/38705